chat.go

```diff
@@ -13,6 +13,25 @@ import (
 // about a chat command that's taking long to execute.
 var ChatCmdTimeout = 10 * time.Second
 
+// Exec handles a string as if it where send by a client.
+// string is imediate returnvalue of chatcommand (if applicable)
+// bool   is if msg is getting forwarded to the minetest server
+func (cc *ClientConn) Exec(msg string) (string, bool) {
+	pkt := &mt.ToSrvChatMsg{
+		Msg: msg,
+	}
+
+	str, fwd := onChatMsg(cc, pkt)
+
+	if fwd {
+		cc.server().Send(*&mt.Pkt{
```

Comment:
SendCmd does the exact same thing
Also dafuq is this *& stuff?!
Btw if SendCmd doesn’t exist, implement the wrapper for mt.Peer.SendCmd yourself